sbol_utilities/package.py

```diff
@@ -64,26 +64,27 @@ def regularize_package_directory(dir: str):
         raise ValueError(f'Package {dir}: {PACKAGE_DIRECTORY} subdirectory should not have any subdirectories of its '
                          f'own, but found {package_sub_dirs[0]}')
 
-def define_package(doc: sbol3.Document):
-    """Function to take an sbol document, check if it is a package, and create
-    a new sbol document with the package definition included
+def define_package(*args: sbol3.Document):
```

Comment:
I believe you don't need to call the variable args --- you could use call it *docs instead
See: https://stackoverflow.com/questions/53676249/are-python-args-and-kwargs-ever-named-something-else-in-practice

Reply:
I also think it will be important to distinguish between the first document, which will be for the package, and the rest of the documents, which will be for sub-packages.

Reply:
I am now passing the first document and the sub-package documents separately, but I immediately combine them all into one list and treat them all the same. When aggregating multiple files, do I need to be treating the root package file differently?

Reply:
Yes, this was creating a bug. I've added an explanatory comment both in the code here and in the relevant .nt file, which illustrates the problem well. Right now, you add the package as a sub-package of itself, which both creates a loop and also doesn't store the members.